frontend/common/Login.tsx

```diff
@@ -45,17 +68,32 @@ export function Login(props: LoginProps) {
   return (
     <ErrorBoundary message={translations.errorText}>
       <Wrapper>
-        <Inner>
-          <Heading headingLevel="h1" size={TitleSizes['2xl']}>
-            {t('Welcome to {{productName}}', { productName })}
-          </Heading>
-          <LoginForm
-            apiUrl={props.apiUrl}
-            authOptions={props.authOptions}
-            onLoginUrl={props.onLoginUrl}
-            hideInputs={props.hideInputs}
+        <LeftHalf>
+          <Inner>
+            <Heading headingLevel="h1" size={TitleSizes['2xl']}>
+              {t('Welcome to {{productName}}', { productName })}
+            </Heading>
+            <LoginForm
+              apiUrl={props.apiUrl}
+              authOptions={props.authOptions}
+              onLoginUrl={props.onLoginUrl}
+              hideInputs={props.hideInputs}
+            />
+          </Inner>
+        </LeftHalf>
+        <RightHalf>
+          <ServiceLogo
+            src={
+              process.env.PRODUCT === 'Automation Hub'
```

Comment:
What about we create a data structure to hold those strings, either an object or an enum?

Something like 


```tsx
const projectNames = {
  ansible: 'Ansible',
  automationHub: 'Automation Hub',
  eventDrivenAutomation: 'Event Driven Automation',
  awx: 'AWX',
};
```

Reply:
We don't want to use that background image anymore. The background should be a solid color.
dark mode: #151515
light mode: #F0F0F0

Reply:
hmm.. I think that the data coming from "process.env.PRODUCT" is already human readable- it's the translation to the logo.svg that I can translate.